nodogsplash/Makefile

```diff
@@ -40,6 +40,7 @@ define Package/nodogsplash/description
 endef
 
 define Package/nodogsplash/install
+	$(CP) ./files/* $(1)/
```

Comment:
Uhh, this is not good.  The PR was draft and then it was merged. 
I would like to avoid this. You can not use CP for all. The previous solution was better, not the best.

Reply:
So you want each file to be referenced?